server/testutils/factories/booking.ts

```diff
@@ -1,15 +1,76 @@
 import { Factory } from 'fishery'
 import { faker } from '@faker-js/faker/locale/en_GB'
+import { addDays } from 'date-fns'
 
 import type { Booking } from 'approved-premises'
 import arrivalFactory from './arrival'
 
-export default Factory.define<Booking>(() => ({
+const today = new Date(new Date().setHours(0, 0, 0, 0)).toISOString()
```

Comment:
It took me a while to work out what this is doing. 
I suggest we use [startOfToday](https://date-fns.org/v2.28.0/docs/startOfToday) from `date-fns` for clarity. 
Or we could save the `new Date().setHours(0, 0, 0, 0)` as a variable called `startOfToday` and use that in the second `new Date()`